health/micro-ui-react19/web/packages/css/src/pages/employee/index.scss

```
@@ -1593,6 +1600,8 @@ tbody {
 
 .digit-popup-heading{
   color:var(--digitv2-lightTheme-primary-2) !important;
+  overflow-wrap: break-word;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)